testbed/gtk/flutter/gtk_plugin_registrant.cc

```diff
@@ -3,9 +3,12 @@
 
 #include "gtk_plugin_registrant.h"
 
+#include <color_panel_plugin.h>
+
 void fl_register_plugins(FlPluginRegistry* registry) {
-  // Example plugin code:
-  // g_autoptr(FlPluginRegistrar) {name}_registrar =
-  // fl_plugin_registry_get_registrar_for_plugin(registry, "{class}");
-  //{name}_panel_plugin_register_with_registrar({name}_registrar);
+  g_autoptr(FlPluginRegistrar) color_panel_registrar =
+      fl_plugin_registry_get_registrar_for_plugin(registry, "ColorPanelPlugin");
+  FlColorPanelPlugin* color_panel_plugin =
+      fl_color_panel_plugin_new(color_panel_registrar);
+  g_object_unref(color_panel_plugin);
```

Comment:
Since the constructor is now public `color_panel_plugin_register_with_registrar()` seems redundant, so I just did the above. Does this match what you expected?

Reply:
I would much rather follow the pattern used on all the other platforms, which is a factory method. I don't see a good reason to diverge from the other platforms on this.